src/plugins/expressions/common/execution/execution.ts

```diff
@@ -220,6 +220,10 @@ export class Execution<
         return createError({ message: `Function ${fnName} could not be found.` });
       }
 
+      if (fn.disabled) {
+        return createError({ message: `Function ${fnName} is disabled.` });
```

Comment:
I wonder if we should be running these through i18n as they are automatically surfaced in the UI on the client?